source/configuration_manual/namespace/index.rst

```diff
@@ -32,6 +32,182 @@ you to update an existing namespace (like how ``15-mailboxes.conf`` does) or
+
+- Default: <empty>
+- Values: :ref:`string`
+
+Name of the namespace.
+Giving name is optional, but doing so enables referencing the configuration later on.
+
+
+.. _setting-namespace_disabled:
+
+``namespace/disabled``
+----------------------
+
+ - Default: `no`
+ - Values: ref:`boolean`
+
+When ``yes`` namespace is disabled, and cannot be accessed by user in any way.
+
+
+.. _setting-namespace_hidden:
+
+``namespace/hidden``
+--------------------
+
+ - Default: `no`
+ - Values: ref:`boolean`
+
+When ``yes`` namespace will be hidden from user.
+It is still accessible if folders are directly opened, or referenced by other components.
+Usually used for lazy expunge or virtual folders.
+
+
+.. _setting-namespace_ignore_on_failure:
+
+``namespace/ignore_on_failure``
+-------------------------------
+
+ - Default: `no`
+ - Values: ref:`boolean`
+
+If namespace :ref:`setting-namespace_location` fails to load, this namespace will not be included.
+
+
+.. _setting-namespace_inbox:
+
+``namespace/inbox``
+-------------------
+
+ - Default: `no`
+ - Values: ref:`boolean`
+
+When ``yes`` this namespace will be considered the one holding ``INBOX`` folder.
+There can be only one namespace like this.
+
+
+.. _setting-namespace_list:
+
+``namespace/list``
+------------------
+
+ - Default: `yes`
+ - Values: ref:`boolean`
+
+Whether to include this namespace in LIST output.
+See also :ref:`setting-namespace_hidden`.
+
+
+.. _setting-namespace_location:
+
+``namespace/location``
+----------------------
+
+ - Default: :ref:`setting-mail_location`
+ - Values: ref:`string`
+
+Specifies driver and parameters for physical mailbox storage.
+
+Example::
+
+  namespace {
+    location = sdbox:/archive/%u
+  }
+
+
+.. _setting-namespace_order:
+
+``namespace/order``
+-------------------
+
+ - Default: `0`
+ - Values: ref:`uint`
+
+Sets display order in IMAP ``NAMESPACES`` command.
+Automatically numbered when untouched.
+
+
+.. _setting-namespace_prefix:
+
+``namespace/prefix``
+--------------------
+
+ - Default: <empty>
+ - Values: ref:`string`
+
+Specifies prefix for namespace.
+Must end with :ref:`hierarchy separator <setting-namespace_separator>`.
+
+Example::
+
+  namespace {
+    prefix = Shared/
+    separator = /
+  }
+
+
+.. _setting-namespace_separator:
+
+``namespace/separator``
+-----------------------
+
+ - Default: <empty>
+ - Values: ref:`string`
+
+Specifies the separator for namespace. This is one character, usual separators used are ``.``, ``/`` and ``^``.
+Recommended separator for new installations is ``/``.
+
+Example::
+
+  namespace {
+    separator = /
+  }
+
+
+.. _setting-namespace_subscriptions:
+
+``namespace/subscriptions``
+---------------------------
+
+ - Default: ``yes``
+ - Values: ref:`boolean`
+
+Whether subscriptions are stored in this namespace.
+This is usually set to no, when the namespace driver cannot store subscriptions for some reason.
+They are then stored in first parent namespace that has this setting set to ``yes``.
```

Comment:
I think better to give example: This is usually set to `no` for shared namespaces so that the shared folders' subscriptions are stored in the user's primary subscriptions file. When set to `no`, the subscriptions are stored in the first parent namespace (based on the prefix) that has this setting set to ``yes``.